anotherfile.js

```diff
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
+    case "add":
+      return a + b;
+    case "subtract":
+      return a - b;
+    case "multiply":
+      return a * b;
+    case "divide":
+if (b === 0) throw new Error("Cannot divide by zero");
```

Comment:
Improve error handling by using a try-catch block to handle division by zero.

```suggestion
function calculate(operation, a, b) {
  try {
    switch (operation) {
      case "add":
        return a + b;
      case "subtract":
        return a - b;
      case "multiply":
        return a * b;
      case "divide":
        if (b === 0) throw new Error("Cannot divide by zero");
        return a / b;
      default:
        throw new Error("Unknown operation");
    }
  } catch (error) {
    console.error('Calculation error:', error);
    return `Error: ${error.message}`;
  }
}
```

**Original code:**
```
if (b === 0) throw new Error("Cannot divide by zero");
```